internal/journald/journald.go

```diff
@@ -0,0 +1,48 @@
+/*
+Copyright (c) Edgeless Systems GmbH
+
+SPDX-License-Identifier: AGPL-3.0-only
+*/
+
+/*
+Package journald provides functions to read and collect journald logs.
+*/
+package journald
+
+import (
+	"context"
+	"os/exec"
+)
+
+type collectionCommand interface {
+	executeCommand() ([]byte, error)
+}
+
+// Command represents a command that is executed by journalctl.
+type Command struct {
```

Comment:
A package should expose an abstraction to its caller. We should name this after what it does, I suggest `Collector`. The caller doesn't need to know if we execute a command underneath or write all those logs by hand... :wink: